kuma/wiki/tests/conftest.py

```diff
@@ -70,3 +118,107 @@ def trans_doc(create_revision, wiki_user):
 @pytest.fixture
 def trans_revision(trans_doc):
     return trans_doc.current_revision
+
+
+@pytest.fixture
+def multi_generational_docs(wiki_user):
+    great_grandparent_doc = Document.objects.create(
+        locale='en-US',
+        slug='Great-grandparent',
+        title='Great-grandparent Document'
+    )
+    Revision.objects.create(
+        document=great_grandparent_doc,
+        creator=wiki_user,
+        content='<p>Great-grandparent...</p>',
+        title='Great-grandparent Document',
+        created=datetime(2017, 4, 24, 13, 49)
+    )
+
+    grandparent_doc = Document.objects.create(
+        locale='en-US',
+        slug='Grandparent',
```

Comment:
The document slugs should reflect the hierarchy - ``Great-grandparent/Grandparent``, ``Great-grandparent/Grandparent/Parent``.  Which may include changing the slugs, since they will quickly get ridiculous. ``DocumentZoneTests`` doesn't do this, but we should, and it's a [problem in production](https://bugzilla.mozilla.org/show_bug.cgi?id=1311142#c1) that may be forbidden by future code changes.